client/consensus/babe/src/lib.rs

```diff
@@ -678,6 +678,7 @@ where
 					chain_head_slot,
 					self.client.info().finalized_number,
 					slot_number,
+					self.logging_target(),
```

Comment:
Not the most elegant maybe, is there a cleaner way @bkchr ?

Reply:
Yeah not that nice, but yeah :shrug: 

@andresilva WDYT?

Reply:
The other option would be to pass the logging target to the strategy itself when creating it but I think that's as nice as this. The reason we use the same logging target here as block production is to make sure these logging messages don't go unnoticed (i.e. people would forget to add some "slots" target). In this case the message is being logged under info so it should always be printed regardless of the target we provide.

Reply:
So you're saying might as well drop the `logging_target` altogether?

Reply:
Either keep the changes you made or drop it altogether. I don't have a strict opinion about this, I guess we can keep it as is.